mysql-test/t/percona_encrypt_tmp_files.test

```diff
@@ -0,0 +1,104 @@
+# Row binlog format to fill binlog cache faster
+--source include/have_binlog_format_row.inc
```

Comment:
The test will be just skipped if the binlog format happens to be different (main suite does not use binlog format combinations) - maybe set the format in the testcase instead?

Reply:
Reworked